Source/Core/VideoBackends/OGL/VertexManager.cpp

```diff
@@ -156,8 +158,20 @@ void VertexManager::vFlush()
   // setup the pointers
   nativeVertexFmt->SetupVertexPointers();
 
+  if (!g_Config.backend_info.bSupportsFragmentStoresAndAtomics && ::BoundingBox::active)
+  {
+    glEnable(GL_STENCIL_TEST);
+    OGL::BoundingBox::stencil_updated = true;
```

Comment:
> Given this is the only place these are used externally, it would probably be preferable to make this value setting part of BoundingBox's API via a function, rather than exposing public variables. (e.g. BoundingBox::UpdateStencilState or something more appropriate if you know of one).

Good call. I moved the variables and logic behind a `BoundingBox::StencilWasUpdated` function (appropriately documented). Does that work for you?

> Reading this again, it's somewhat unfortunate this is directly within the vertex manager. BoundingBox could have been a class with no static behavior, and just have an instance of itself situated within the OGL Renderer class otherwise.

Yeah, I couldn't think of a better place to put this logic. It could conceivably fit in `ProgramShaderCache` somewhere (since that's the other dependency being used here), but that seemed nonintuitive to me.